README.md

```diff
@@ -497,7 +506,8 @@ sudo docker-compose up -d
 ```
 
 The four differences are:
```

Comment:
```suggestion
The only differences are:
```
(there's seven! and we probably don't want to update the number every time we change this)

Reply:
Good catch, but it’s not exhaustive either.
```suggestion
The main differences are:
```